.github/styles/Google/MarkdownLinks.yml

```diff
@@ -0,0 +1,7 @@
+extends: conditional
+message: "Remember to include the link."
+level: error
+ignorecase: false
+# Ensures that the existence of 'first' implies the existence of 'second'.
+first: '(?:\[[\w\\`",.() -]+\])(\[[\w-]+\])'
+second: '(\[[\w-]+\])(?:: [\w\/.&#=:?-]+$)'
```

Comment:
Fixed these and tested (realized I was missing a lot of possible link text and URL symbols). Here are the test results @Loquacity @jacobprall . Does that look good to you or am I missing some cases?

![152819646-341b6e36-7469-4808-8855-bdd4e7ed82e2](https://user-images.githubusercontent.com/26616127/152822704-55f2f9fd-611a-440b-a5db-892ba23414e8.png)
![152821881-fb996a93-5269-4d1d-902c-293e5c6a88dc](https://user-images.githubusercontent.com/26616127/152822711-a3465b23-5f2c-480c-ad1e-d5e37c380efe.png)

Reply:
LGTM!

Reply:
🚢 it!